test/pkg/payload/get_entries.go

```diff
@@ -28,7 +28,11 @@ func GetEntries(yamlfile map[string]string, targetNS, targetBranch, targetEvent
 	for target, file := range yamlfile {
 		name := strings.TrimSuffix(filepath.Base(target), filepath.Ext(target))
 		// add some random character to name so that each PR has different name
-		extraParams["PipelineName"] = name + "-" + strings.ToLower(random.AlphaString(4))
+		if targetEvent == "incoming" {
+			extraParams["PipelineName"] = name
```

Comment:
what does this do? generally if there is comment above, it would be good to update it if you change what it does...